### ISSUE TITLE
Include license file in source distribution

### ISSUE THREAD

**Issue by synapticarbors** [Cadair/matrix_api_async#1]
It would be helpful to include the license file in the source distribution

**Comment by Cadair** [comment]
yep I should do that, although at this point I would highly recommend people use https://github.com/poljar/matrix-nio over this where practical. I should add that to the readme :grin:

**Comment by synapticarbors** [comment]
I'm just trying to get `opsdroid` up-to-date, and had to make a conda recipe for this package (https://github.com/conda-forge/staged-recipes/pull/11930).
